src/libraries/System.Numerics.Vectors/ref/System.Numerics.Vectors.cs

```diff
@@ -153,6 +153,7 @@ public partial struct Quaternion : System.IEquatable<System.Numerics.Quaternion>
         public float Z;
         public Quaternion(System.Numerics.Vector3 vectorPart, float scalarPart) { throw null; }
         public Quaternion(float x, float y, float z, float w) { throw null; }
+        public static System.Numerics.Quaternion Zero { get { throw null; } }
```

Comment:
can you please generate this file? that will make it ordered, so next time we generate there won't be extra diffs. info:
https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/docs/coding-guidelines/updating-ref-source.md#L16